### PR TITLE
Release: 06-14 (main)

### PR DESCRIPTION
**Pull request by taylorjdawson**
### PRs
- #1031 
- #1039 
- #1056 
- #1057 
- #1060 
- #1062
- #1063 
### Issues
- #955 
- #1061 
- #1030

### CODE REVIEW

File: packages/core/src/validation.ts [hunk 1]
```diff
@@ -14,14 +14,30 @@ const chainId = Joi.string().pattern(/^0x[0-9a-fA-F]+$/)
 const chainNamespace = Joi.string().valid('evm')
 const unknownObject = Joi.object().unknown()
 
+// const address = Joi.string().regex(/^0x[a-fA-F0-9]{40}$/)
```
Comment (Adamj1232) [inline packages/core/src/validation.ts:17]: Should this be removed?

File: packages/walletconnect/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/walletconnect",
-  "version": "2.0.1",
+  "version": "2.0.2-alpha.1",
```
Comment (Adamj1232) [inline packages/walletconnect/package.json:3]:
```suggestion
  "version": "2.0.2",
```

File: packages/core/src/validation.ts (hunk 1 above)
Comment (Adamj1232) [inline packages/core/src/validation.ts:17]:
```suggestion
```

**Review by Adamj1232 — APPROVED** [review]
Looks good LGTM!!

**Review by mahmud-bn — APPROVED** [review]

**Pull request merged**